code/datums/stars.dm

```diff
@@ -101,6 +101,11 @@
 			no_unload_reason = "RELAY"
 			return 0
 
+	for(var/obj/machinery/computer/shuttle/fob/F in GLOB.machines)
+		if((F.z in z_levels))
+			no_unload_reason = "FOB SHUTTLE"
+			return 0
```

Comment:
instead of this make the fob landing activate keep_loaded

Reply:
no_unload_reason is displayed on the starmap. This indicates where the fob shuttle is. See a few lines above where tcomms relays do the same thing.

Reply:
Although I should make it not show on the helms console if the FOB is docked with the main ship still.